go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/siderolabs/gen
 
-go 1.19
+go 1.21.0 // Starting with Go 1.21 you have to provide the third digit too.
```

Comment:
wow

Reply:
Are you sure about this - my [personal project](https://github.com/utkuozdemir/pv-migrate/blob/master/go.mod#L4) accepted 1.21 without issues.